tests/unit/test_init.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import unittest
+from argparse import Namespace
+from unittest.mock import MagicMock, Mock, patch
+
+import magma_access_gateway_installer
+
+
+class TestAGWInstallerInit(unittest.TestCase):
+    def setUp(self) -> None:
+        pass
+
+    def test_given_agw_installer_init_when_cli_arguments_are_given_then_parser_parses_arguments_correctly(  # noqa: E501
+        self,
+    ):
+        args_parser = magma_access_gateway_installer._cli_arguments_parser(
+            ["--ip-address", "1.2.3.4/24", "--gw-ip-address", "2.3.4.5"]
```

Comment:
I think IP's can be variable, just like in `test_agw_network_configurator.py` where we have variables for `test_static_ip_address`.

Reply:
Done.